iocStats/install.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+##########################################################################
+##### boilerplate install script for support modules #####################
+##########################################################################
+
+# ARGUMENTS:
+#  $1 VERSION to install (must match repo tag)
+
+# get the name of this folder which is the same as name of the support module
+NAME=$(basename $(dirname ${0}))
+VERSION=${1}
+
+IBEK_SUPPORT=$(realpath $(dirname ${0})/..)
+source ${IBEK_SUPPORT}/_global/functions.sh
+
+git_clone_tag ${NAME} ${VERSION}
+
+support add-module-to-release ${NAME}
+
```

Comment:
Or should we have
```sh
add_ioc_libs devIocStats
add_ioc_dbds devIocStats.dbd
```

Reply:
I MUCH prefer this. Forgot that I wanted to review this bit before doing the PR.

Reply:
This is now implemented